guides/embedders/mistral.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Semantic Search with Mistral Embedding - Meilisearch documentation
+description: This guide will walk you through the process of setting up Meilisearch with Mistral embeddings to enable semantic search capabilities.
+---
+
+# Semantic Search with Mistral Embedding
+
+## Introduction
+
+This guide will walk you through the process of setting up Meilisearch with Mistral embeddings to enable semantic search capabilities. By leveraging Meilisearch's AI features and Mistral's embedding API, you can enhance your search experience and retrieve more relevant results.
+
+## Requirements
+
+To follow this guide, you'll need:
+
+- A Meilisearch Cloud project running version 1.6 or above with the Vector store activated.
+- A Mistral account with an API key for embedding generation. You can sign up for a Mistral account at [Mistral](https://mistral.ai/).
+- No backend required.
+
+## Explanation
```

Comment:
This is great, but I don't think it's useful in this guide. We can make the guide more straightforward by removing this section IMO.

I would recommend updating the intro to include links to the relevant "explanation" pages, and removing this section.

Reply:
The full section explanation?

Reply:
Will do it. However I can redirect to this page https://www.meilisearch.com/docs/learn/ai-powered-search/difference_full_text_ai_search to know more but there is nothing exposing really the capabilities and the benefits of using Meilisearch.

Reply:
Let's leave the explanation in your article for now then. We will extract it to some dedicated page later on.